.travis.yml

```diff
@@ -1,9 +1,24 @@
 language: ruby
 rvm:
+- 2.2
 - 2.1
-- 2.0.0
-- 1.9.3
+- 2.0
 before_script: bundle update
 script: "script/cibuild"
 sudo: false
 cache: bundler
+
+matrix:
+  include:
+    - # GitHub Pages
+      rvm: 2.1.1
+      env: GH_PAGES=true
+    - # Ruby 1.9
+      rvm: 1.9
+      env: JEKYLL_VERSION=2
+
+env:
+  matrix:
+    - ""
+    - JEKYLL_VERSION=3.0.0.beta8
+    - JEKYLL_VERSION=2
```

Comment:
`~> 2` allows `3`, so it won't lock to 2.x. It would be better to say `2.0` here, as `~> 2.0` only allows 2.x.